### PR TITLE
Fix/SFINT-3128 es6 is now 100% es6

### PR DESCRIPTION
**Pull request by jeremierobert-coveo**
Everything is in the title.

### CODE REVIEW

**Comment by jeremierobert-coveo** [comment]
C'est pas sense toucher la partie commonjs 😛

File: config/webpack.config.js [hunk 1]
```diff
@@ -29,7 +29,10 @@ module.exports = {
                 test: /\.ts$/,
                 loader: 'ts-loader',
                 options: {
-                    configFile: path.resolve('./config/tsconfig.json')
+                    configFile: path.resolve('./config/tsconfig.json'),
+                    compilerOptions: {
+                        target: 'es5'
```
Comment (tedre191) [inline config/webpack.config.js:34]: Not sure why you changed the tsconfig.json to use ES6, only to force es5 here ?

Comment (jeremierobert-coveo) [reply config/webpack.config.js:34]: The es6 build will build es6 instead of es5

File: src/utils/icons.ts [hunk 2]
```diff
@@ -1,5 +1,3 @@
-/// <reference path='../typings/custom.d.ts'/>
```
Comment (louis-bompart) [inline src/utils/icons.ts:1]: Pretty sure typescript was freaking out last time I tried 🤔

**Pull request merged**